frontend/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,46 @@
+type TagProps = {
+  /** Content of tag */
+  children: React.ReactNode;
+
+  /** href destination Link */
+  to: String;
```

Comment:
In Typescript immer die klein geschriebenen Typen nutzen: Anstatt `String` => `string`. Dann geht das unten auch mit href  direkt `href={to}` :D